### ISSUE TITLE
Day 5 - tap gesture not triggered

### ISSUE THREAD

**Issue by thisIsTheFoxe** [satoshi0212/visionOS_30Days#4]
Hi,
Awesome project! Helped me a lot getting into and familiar with VisionPro and the SDK. However, one thing I struggled with was that all my `SpatialTapGesture()` didn't work. So, I looking into [Day 5](https://github.com/satoshi0212/visionOS_30Days/blob/69e85e135403970fa453f69253e3d245e49b55ca/Day5/Day5/ImmersiveView.swift#L21) and it wasn't working there either.. I eventually found [Day 13](https://github.com/satoshi0212/visionOS_30Days/blob/69e85e135403970fa453f69253e3d245e49b55ca/Day13/Day13/ViewModel.swift#L11-L12) and [this WWDC video](https://developer.apple.com/videos/play/wwdc2023/10082/?time=1340) which cleared things up a lot..
TL;DR, you need to set `InputTargetComponent` & `CollisionComponent`, which isn't being done for `contentEntity`.

So, I'd suggest either removing the `.gesture` or setting the components. Otherwise it's a bit confusing, at least imo.. ^^
